packages/fether-react/src/utils/estimateGas.js

```diff
@@ -90,10 +95,15 @@ export const txForErc20 = (tx, token) => {
  * passed to post$(tx).
  */
 export const txForEth = tx => {
-  return {
+  let output = {
```

Comment:
I'm a bit confused with that one, I read here and [there](https://evertpot.com/javascript-let-const/) that `const` should be used when a variable will not be reassigned. I chose `let` in this case because `output` will change.

Reply:
It's about JS's memory management. When you define `const output = ...`, the variable `output` has a memory address, and that address cannot be changed (i.e. you cannot do `output = somethig_else`). However, you can attach other variables that point to this intial address using `output.gas = ...`. 

With `let output = ...`, you can do `output = somethign_else`, i.e. change the address of the the variable `output` in memory.

Note: it's not exactly the same in Rust, that might be the confusion.